src/components/profile/CameraButton.js

```diff
@@ -1,32 +1,32 @@
 // @flow
 import React from 'react'
 import normalize from 'react-native-elements/src/helpers/normalizeText'
-import IconButton from 'react-native-paper/src/components/IconButton'
+import Icon from 'react-native-elements/src/icons/Icon'
 import { StyleSheet, View } from 'react-native'
 
 const CameraButton = ({ handleCameraPress, containerStyles = {}, iconStyles = {} }) => (
   <View style={[cameraStyles.container, containerStyles]}>
-    <IconButton
+    <Icon
       onPress={handleCameraPress}
-      size={normalize(24)}
-      color="white"
-      icon="photo-camera"
-      style={[cameraStyles.icon, iconStyles]}
+      size={normalize(20)}
+      color="#0C263D"
```

Comment:
Shouldn't this be part of the `theme`? (`theme.colors.darkBlue`)

Reply:
Done